php/razorpay_js/index.html

```diff
@@ -0,0 +1,402 @@
+				}
+				/**
+				 * @note this will create a new customer in razorpay app
+				 */
+				let newRazorpayCustomer = await createRazorpayCustomer({
+					"name": `${chargebeeCustomerDetail.first_name || ''} ${chargebeeCustomerDetail.last_name || ''}`,
+					"email": chargebeeCustomerDetail.email || null
+				});
+				/**
+				 * @note For existing razorpay customers, skip the above part and pass the respective customer details
+				 */
+				console.info(
+					`======== Customer Details ==========`
+				);
+				console.info(newRazorpayCustomer);
+				let orderDetails;
+				if (newRazorpayCustomer.id) {
+					orderDetails = await createOrder(newRazorpayCustomer.id, estimationAmount, currencyCode);
+					console.info(
+						`=========== Order Created ===========`
+					);
+					console.info(orderDetails);
+					if (orderDetails && orderDetails.id) {
+						console.info(
+							`========== Inintating Razorpay Checkout ==========`
+						);
+						openRPCheckout(
+								newRazorpayCustomer,
+								orderDetails.id,
+								orderDetails.amount,
+								orderDetails.currency
+						);
+					}
+				}
+				console.info(
+				"========= Payment Done =========== "
+				);
+			} catch (err) {
+				payButton.classList.remove('disable')
+				console.log("Error:", err);
+			}
+		}
+
+		async function createChargebeeCustomer(customerDetail) {
+			console.info(
+					"====== Creating CB Customer =========="
+			);
+			let myHeaders = new Headers();
+			myHeaders.append("Content-Type", "application/x-www-form-urlencoded");
+
+			let urlencoded = new URLSearchParams();
+			urlencoded.append("first_name", customerDetail.first_name || '');
+			urlencoded.append("last_name", customerDetail.last_name || '');
+			urlencoded.append("email", customerDetail.email || '');
+
+			let requestOptions = {
+				method: 'POST',
+				headers: myHeaders,
+				body: urlencoded,
+			};
+
+			let newCbCustomer = await fetch("http://localhost:3000/api/create_cb_customer", requestOptions)
+					.then(response => response.json())
+					.then(result => result)
+					.catch(error => console.log('error', error));
+			return newCbCustomer;
+		}
+
+		async function createCbEstimate() {
+			console.info(
+				"====== Creating CB Subscription for Items Estimation =========="
+			);
+			try {
+				let myHeaders = new Headers();
+				myHeaders.append("Content-Type", "application/x-www-form-urlencoded");
+
+				let urlencoded = new URLSearchParams();
+				urlencoded.append(
+					"subscription_items[item_price_id][0]",
+					ITEM_PRICE_ID
+				);
+				urlencoded.append("subscription_items[billing_cycles][0]", 2);
+				urlencoded.append("subscription_items[quantity][0]", 1);
+				urlencoded.append("cb_customer_id", CB_CUSTOMER_ID);
+
+				let requestOptions = {
+					method: "POST",
+					headers: myHeaders,
+					body: urlencoded,
+				};
+
+				let estimate = await fetch(
+					"http://localhost:3000/api/create_estimate",
```

Comment:
Extract the server URL out to a global variable, then build the URLs while making the request